src/test/java/org/jvnet/jenkins/plugins/nodelabelparameter/parameterizedtrigger/NodeLabelBuildParameterTest.java

```diff
@@ -23,6 +23,11 @@
  */
 package org.jvnet.jenkins.plugins.nodelabelparameter.parameterizedtrigger;
 
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
The rest of the plugin uses JUnit 4 assertions.  Please stay with JUnit 4 assertions in this file as well
```suggestion
```